pkg/konfluxgen/kustomize/fbc-builder.yaml

```diff
@@ -94,10 +72,24 @@ spec:
     description: Build a source image.
     name: build-source-image
     type: string
-  - default: "false"
+  - default: "true"
     description: Add built image into an OCI image index
     name: build-image-index
     type: string
+  - default: []
+    description: Array of --build-arg values ("arg=value" strings) for buildah
+    name: build-args
+    type: array
+  - default: ""
+    description: Path to a file with build arguments for buildah, see https://www.mankier.com/1/buildah-build#--build-arg-file
+    name: build-args-file
+    type: string
+  - default:
+    - linux/x86_64
+    description: List of platforms to build the container images on. The available
+      set of values is determined by the configuration of the multi-platform-controller.
+    name: build-platforms
+    type: array
```

Comment:
We would need to pass this parameter in the PipelineRun:

https://github.com/openshift-knative/hack/blob/main/pkg/konfluxgen/pipeline-run.template.yaml#L49-L56

so the check would become something like this:

```
    - name: build-platforms
      value:
        - linux/x86_64
    {{{- if eq .Pipeline "fbc-builder" }}}
    - name: skip-fbc-related-image-check
      value: "true"
    {{{- end }}}
```

Reply:
Good point